frontend/src/components/regular/ui_schema_wrapper.vue

```diff
@@ -1,39 +1,13 @@
 <template>
-
-  <div @mouseover="mouseover"
-       @mouseleave="mouseleave"
-       v-if="visible"
-       >
-
-    <slot>
-
-    </slot>
-
+  <div 
+    v-if="visible"
+    @mouseover="mouseover"
+  >
+    <slot />
   </div>
 </template>
 
 <script lang="ts">
-
-/*
- 
- *
- *  EXAMPLE USAGE:
```

Comment:
why remove example usage? is it in docs now?

Reply:
The idea is to keep this on Storybook, will show you on the call

Reply:
Onboard for storybook!